test/src/parsers/HTMLLexer.js

```diff
@@ -139,6 +139,38 @@ describe('HTMLLexer', function() {
     ].forEach(testParse.bind(this, 'label'));
   });
 
+  it('should parse tags with one attribute', function() {
+    [
+        '<div name="TestDiv1" />',
+        '<label id=3 />',
+        '<pre class="idl">Some stuff here</pre>'
+    ].forEach(function(str) {
+      var value = testParse('htmlPart', str);
+      if ( ! value ) return;
+
+      expect(Array.isArray(value.attributes)).toBe(true);
+      expect(value.attributes.length).toBe(1);
+      expect(value.attributes[0]).toBeDefined();
+    });
+  });
+
+  it('should parse tags multiple attributes', function() {
```

Comment:
I think we should have more tests, each that tests one thing (or a particular combination of things). Maybe something like:

- Multiple attributes
- Attribute without value
- Attribute without quotes
- Attribute in single quotes
- Attribute with `&name;` escape of some kind
- Attribute containing whitespace (maybe include ` \t\r\n`)